src/main/java/de/tum/in/www1/artemis/service/FilePathService.java

```diff
@@ -40,6 +40,10 @@ public static String getExamUserSignatureFilePath() {
         return Path.of(fileUploadPath, "images", "exam-user", "signatures").toString();
     }
 
+    public static String getSlideImageFilePath() {
+        return Path.of(fileUploadPath, "attachments", "slides", "attachment-unit").toString();
```

Comment:
> I suggest placing this under `getAttachmentUnitFIlePath()`

How can i place under `getAttachmentUnitFIlePath` function? Or you mean to switch it around -> `"attachments", "attachment-unit", "slides"`  and keep the `getSlideImageFilePath` function?

And the reason i put it like this is that we might have to split attachment as well into single slides in the feature and not only for attachment-units

Reply:
Yes, keep the method. With "place it under" I mean move the method down so it fits in with the other attachment paths.

The reason I suggest this is because you typically want to go from big to small.

So technically, we should even to `attachment-unit` -> `attachments` -> `slides`? I'm not quite sure here.

Reply:
i switched it around. Now its `attachments/attachment-unit/{id}/slide/{slideNumber}`